### ISSUE TITLE
Save File name should be the Process or Collaboration name by default

### ISSUE THREAD

**Issue by StephenOTT** [camunda/camunda-modeler#1092]
When saving a BPMN, the modler should default to using the `id` value when it is a single BPMN, and when there are multiple pools, it uses the collaboration id.

have `diagram-1.bpmn` is really useless ;)

**Comment by nikku** [comment]
If you're interested, open a quick poll in the [Camunda form](https://forum.camunda.org) and see how whether there exists a _common_ naming convention.

**Comment by StephenOTT** [comment]
Agreed on the collaboration point. Not to many. Can't think of a time that I have. But it's better the generic "diagram".  And ignoring collabs: when not using pools, it's a really big nice to have to not have to rewrite the same content in the save dialogue.
